actors/miner/tests/move_partitions_test.rs

```diff
@@ -627,3 +556,77 @@ fn fault_and_expire_after_move() {
 
     h.check_state(&rt);
 }
+
+#[test]
+fn dispute_after_move() {
+    let (mut h, rt) = setup();
+
+    let sectors_info = h.commit_and_prove_sectors(
+        &rt,
+        1,
+        DEFAULT_SECTOR_EXPIRATION,
+        vec![vec![10], vec![20]],
+        true,
+    );
+    h.advance_and_submit_posts(&rt, &sectors_info);
+
+    let st = h.get_state(&rt);
+    let (orig_deadline_id, partition_id) =
+        st.find_sector(&rt.store, sectors_info[0].sector_number).unwrap();
+
```

Comment:
Suggested comment: move a partition from a deadline that still needs WindowPoST verification.